src/Microsoft.Extensions.ApiDescription.Design/Microsoft.Extensions.ApiDescription.Design.csproj

```diff
@@ -42,6 +42,11 @@
       <PackagePath>tools/dotnet-getdocument.dll</PackagePath>
       <StrongName>$(AssemblySigningStrongName)</StrongName>
     </SignedPackageFile>
+    <SignedPackageFile Include="../dotnet-getdocument/bin/$(Configuration)/netcoreapp2.1/publish/dotnet-getdocument.exe">
```

Comment:
@dougbu the build produces an extra exe in the publish dir. Filing https://github.com/aspnet/Mvc/issues/8681 to investigate this later.

Reply:
Sounds good 😺